plugincontainer/container_runner.go

```diff
@@ -135,17 +132,16 @@ func NewContainerRunner(logger hclog.Logger, cmd *exec.Cmd, cfg *config.Containe
 		},
 	}
 
-	if cfg.UnixSocketGroup != "" {
-		containerConfig.Env = append(containerConfig.Env, fmt.Sprintf("%s=%s", plugin.EnvUnixSocketGroup, cfg.UnixSocketGroup))
-		hostConfig.GroupAdd = append(hostConfig.GroupAdd, cfg.UnixSocketGroup)
+	if cfg.GroupAdd != 0 {
+		hostConfig.GroupAdd = append(hostConfig.GroupAdd, fmt.Sprintf("%d", cfg.GroupAdd))
```

Comment:
Very minor suggestion: it's slightly faster (and maybe as readable?) to use `strconv.Itoa(x)` instead of `fmt.Sprintf("%d", x)`.

```suggestion
		hostConfig.GroupAdd = append(hostConfig.GroupAdd, strconv.Itoa(cfg.GroupAdd))
```

Reply:
Oh yeah, good call! Thanks